diff --git a/lib/helpers.js b/lib/helpers.js
index 04b354132b01bf276271c82b0efeff5474a95179..9c64b8774113d471f9e204a9e587adaa55253efd 100644
--- a/lib/helpers.js
+++ b/lib/helpers.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var uri = require('url');
+import * as uri from 'node:url';
 
 var ValidationError = exports.ValidationError = function ValidationError (message, instance, schema, propertyPath, name, argument) {
   if (propertyPath) {
diff --git a/lib/scan.js b/lib/scan.js
index 391c0d2834a68e8a19eb46013a50dc4ac50be99e..ab991cfd1e1c08f766d892d02ce3ae259957a643 100644
--- a/lib/scan.js
+++ b/lib/scan.js
@@ -1,6 +1,6 @@
 "use strict";
 
-var urilib = require('url');
+import * as urilib from 'node:url';
 var helpers = require('./helpers');
 
 module.exports.SchemaScanResult = SchemaScanResult;
diff --git a/lib/validator.js b/lib/validator.js
index ad3eb2630fe1d6c486770dc3e7a3d94e318ef3d3..530332fe6827b9a6bdb26666b5df675c09e978a3 100644
--- a/lib/validator.js
+++ b/lib/validator.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var urilib = require('url');
+import * as urilib from 'node:url';
 
 var attribute = require('./attribute');
 var helpers = require('./helpers');
